src/index.html

```diff
@@ -4,9 +4,353 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
+
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a class="img-logo" href="#">
+        <img src="/src/images/logo.png" alt="Moyo logo">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="#">APPLE</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">SAMSUNG</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">SMARTPHONES</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+            class="nav__link "
+            href="#"
+            data-qa="nav-hover"
+            >
```

Comment:
Don't forget)
![image](https://github.com/mate-academy/layout_catalog/assets/91411692/1aaf8b6e-c24c-4e5f-b99d-68d26dc60af6)

```suggestion
            <a
              class="nav__link "
              href="#"
              data-qa="nav-hover"
            >
```